examples/resources/hpegl_metal_network/README.md

```diff
@@ -96,15 +96,20 @@ The following arguments are supported:
 - `name` - The name of the network.
 - `description` - (Optional) Some descriptive text that helps describe the network and purpose.
 - `location` - Where the network is to be created in country:region:data-center style.
+- `host_use` - (Optional) Host network use policy. Allowed values are Required, Oprional and Default.
+- `purpose` - (Optional) Purpose of the network. Allowed values are Backup, Storage, vmKernel, vmNSX-T, vMotion, vCHA, vmFT, iSCSI-A, iSCSI-B, Telemetry and External
 - `ip_pool` - (Optional) IP pool used by the network. If not defined an IP allocated from the hoster IP pool factory will be used.
+- `vlan` - (Optional) VLAN ID of the network. If not specified, it is allocated from the reserved pool.
+- `vni` - (Optional) VNI ID of the network. If not specified, it is allocated from the reserved pool.
 
 ### Attribute Reference
 
 In addition to the arguments listed above, the following attributes are exported:
 
 - `location_id` - Unique ID of the location.
-- `kind` - The kind of network, e.g. "Custom".
-- `host_use` - The requirement of a host to use this network, e.g. "Required" or "Optional"
+- `ip_pool_id` - Unique ID of the network IP pool.
+- `vlan` - VLAN ID of the network when it is allocated from the reserved pool.
+- `vni` - VNI ID of the network when it is allocated from the reserved pool if required.
```

Comment:
should this be here? this is "additional" right? (already on line 102 and 103)

Reply:
vid and vni are now optional and computed, so in line 102 and 103 we account for the "computed" part i.e. when the user doesn't provide values for them.

Reply:
The first list comprised `arguments` that can be passed via a main.tf while the second list contains `attributes` of the resource that are known after the TF apply. The thing is that VID/VNI are both arguments and attributes, if they are not provided they are computed "exported" from the reserved POD pools., in the TF apply or TF plan they will show up as `(known after apply)`. I think the user should be aware of this.

Reply:
perhaps adding "computed" to line 107 would help convey this better?

Reply:
> perhaps adding "computed" to line 107 would help convey this better?

I agree 107 "exported" is not clear to me as a casual reader. It may be a TF term, but may be better if it reads 
"In addition to the arguments listed above, the following attributes are returned to the user", or "...are computed & returned to the user". But that may become inconsistent w/ other pages?